jibri/rootfs/defaults/jibri.conf

```diff
@@ -0,0 +1,58 @@
+jibri {
+    api {
+        http {
+            external-api-port = {{ .Env.JIBRI_HTTP_API_EXTERNAL_PORT | default 2222 }}
```

Comment:
Instead of using `default`(hardcoded value), I would recommend using env variable in such a way that if it's not configured then it should be referenced from `reference.conf` file.  @bgrozev recommended this approach when I created new jicofo config PR. 
for example. 
```
{{  if .Env.JIBRI_HTTP_API_EXTERNAL_PORT }}
external-api-port = {{ .Env.JIBRI_HTTP_API_EXTERNAL_PORT }}
{{ end }}

```

Reply:
Yes, this is the preferred way: the default value is already encoded in `reference.conf` in the jibri repo, so we don't want another place where a default is set.